README.md

```diff
@@ -1,2 +1,146 @@
-# leaderboard
-A leaderboard connected to an API to display the top scores of a game.
+<div align="center">
+
+  <h1><b>Leaderboard</b></h1>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+**[LB]** is a website to display and add the top scores of a game. 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>Javascript</li>
+    <li>Webpack</li>
+    <li>Gitflow</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Save and display unlimited amounts of tasks]**
+- **[Add and delete functions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: any web-browser.
+
+### Setup
+
+Clone this [repository](https://github.com/maov19/leaderboard) to your desired folder.
+
+### Install
+
+No installation required.
+
+### Usage
+
+To run the project, open index.html in any web browser.
+
+### Run tests
+
+To run tests, open index.html in any web browser.
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- Hey, since you won't be adding a test to this project, please remove this section from the readme to make your readme have the correct content relevant to the project.

Reply:
Updated, thanks!